addon/helpers/consume-context.js

```diff
@@ -2,9 +2,9 @@ import Helper from '@ember/component/helper';
 import { getProviderFor } from '../-private/get-provider-for';
 
 export default class ContextConsumerHelper extends Helper {
-  compute([key]) {
-    const provider = getProviderFor(this, key);
+  compute([key], { fallback }) {
+    const provider = getProviderFor(this, key, fallback);
 
-    return provider.value;
+    return (provider && provider.value) || fallback;
```

Comment:
What do you think about using a ternary here instead? That feels "right" for this use-case, rather than using `&&` and `||`

```suggestion
    return provider ? provider.value : fallback;
```